webapp/sources/rudder/rudder-templates/src/main/scala/com/normation/templates/FillTemplatesService.scala

```diff
@@ -50,7 +50,7 @@ import com.normation.zio.ZioRuntime
 import zio._
 import zio.syntax._
 import com.normation.zio._
-import org.apache.commons.lang.StringUtils
+import org.apache.commons.lang3.StringUtils
```

Comment:
Do you remember why we need to use that here, and only here? Perf reason ?

Reply:
yes, see https://github.com/Normation/rudder/pull/3293/files#diff-5e75cf601f6a1cca6e6f37ba80ee2658d103fe0a3f44fb412967f256dfd43b43L240